packages/flutter/lib/src/material/about.dart

```diff
@@ -211,6 +213,61 @@ void showAboutDialog({
   );
 }
 
+/// Displays an [AboutDialog], which describes the application and provides a
+/// button to show licenses for software used by the application.
+///
+/// The arguments correspond to the properties on [AboutDialog].
+///
+/// If the application has a [Drawer], consider using [AboutListTile] instead
+/// of calling this directly.
+///
+/// If you do not need an about box in your application, you should at least
+/// provide an affordance to call [showLicensePage].
```

Comment:
nit: we should remove this. We do not tell people whether they should do something in the widget docs, especially around something like this.

Reply:
Hi @MitchellGoodwin, I understand your concern, but if you check the documentation for showAboutDialog, specifically lines 173 and 174, one of the contributors has written something similar. If we remove this documentation, should we also consider removing the documentation for showAboutDialog?

Reply:
We can leave it then